guides/common/modules/proc_configuring-rhui-alternate-content-sources.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Configuring_RHUI_Alternate_Content_Sources_{context}"]
+= Configuring RHUI Alternate Content Sources
```

Comment:
What is RHUI? For now, all ACS procedures can be part of all build targets IMHO. In downstream, we don't run orcharhino on Foreman 3.5+ yet.

Reply:
Red Hat Update Infrastructure. It's the service deployed by Red Hat inside clouds such as AWS, Azure and Google to have the Red Hat CDN available locally, without additional bandwidth costs.